pkg/webui/console/components/mac-settings-section/index.js

```diff
@@ -107,6 +126,21 @@ const maxDutyCycleOptions = [
 const encodeAdrMode = value => ({ [value]: {} })
 const decodeAdrMode = value => (value !== undefined ? Object.keys(value)[0] : null)
 
+const getDataRate = data_rate => {
+  const { lora, fsk, lrfhss } = data_rate
+  // The encoding below mimics the encoding of the `modu` field of the UDP packet forwarder.
+  if (lora) {
+    const { bandwidth, spreading_factor } = lora
+    return `SF${spreading_factor}BW${bandwidth / 1000}`
+  } else if (fsk) {
+    const { bit_rate } = fsk
+    return `${bit_rate}`
+  } else if (lrfhss) {
+    const { modulation_type, operating_channel_width } = lrfhss
+    return `M${modulation_type ?? 0}CW${operating_channel_width / 1000}`
+  }
+}
+
```

Comment:
This is a duplicate of `pkg/webui/console/components/events/utils/index.js`. Please move this out into a general utility that both the events and this form can use.